src/libcore/ptr/mod.rs

```diff
@@ -1266,7 +1266,7 @@ impl<T: ?Sized> *const T {
     ///
     /// # Panics
     ///
-    /// This function panics if `T` is a Zero-Sized Type ("ZST").
+    /// This function panics if `T` is a Zero-Sized Type ("ZST")
```

Comment:
Why'd you remove the full stop? Makes more sense to have it IMO.

Reply:
```suggestion
    /// This function panics if `T` is a Zero-Sized Type ("ZST").
```